build/iso-codes/build.sh

```diff
@@ -35,8 +35,8 @@ SUMMARY="ISO code lists and translations"
 DESC="$SUMMARY"
 
 # As of 3.70, iso-codes needs Python3 to build. Make sure python3 is around.
-BUILD_DEPENDS_IPS="omniti/runtime/python-34"
-export PATH=/opt/python34/bin:$PATH
+BUILD_DEPENDS_IPS="ooce/runtime/python-36"
+export PATH=/opt/ooce/bin:$PATH
```

Comment:
Let's make the `/opt/ooce/bin/python` symlink a mediated one then people can choose the python version that they want to use if /opt/ooce/bin is in their path.
We can deal with that in a separate PR though. 

The `iso-codes` build isn't fussy, it just needs a python3 so this change to use /opt/ooce/bin/ seems reasonable to me.